charts/orkes-conductor-community/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: orkes-conductor-community
+description: Orkes Conductor Community Deployment Helm Charts
+
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 1.0.0
```

Comment:
Can you make this version the same version as the conductor binaries?

Reply:
done.